packages/storybook/stories/va-memorable-date-uswds.stories.jsx

```diff
@@ -66,6 +66,7 @@ const CustomValidationTemplate = ({ label, name, required, error, uswds, value }
         name={name}
         required={required}
         error={errorVal}
+        invalidYear={!!errorVal || null}
```

Comment:
This `invalidYear` setting is overridden by the internal `checkIsNaN` and `validation` checks. Adding this here for future testing